src/common/config_parser.cpp

```diff
@@ -562,7 +562,13 @@ void ConfigParser::addOptionsValidation(cli::CLIWrapper& cli) {
       "Additional args passed to --valid-script-path. These are inserted"
       " between the script path and the output translation-file path");
   cli.add<std::string>("--valid-translation-output",
-     "Path to store the translation");
+     "(Template for) path to store the translation\n"
+     "    template parameters:\n"
```

Comment:
Maybe we can start with the simplest usage followed by a more complex one, for example, "Path to store the translation. Can also be a template, e.g. valid-after-{U}-updates-{T}-tokens.out, with parameters: (...)" or similar.

Reply:
Done.